web-api/openapi/front_office.yaml

```diff
@@ -165,6 +200,40 @@ components:
           type: number
         properties:
           $ref: './schemas/document.yml#/DocumentProperties'
+    StatelessPersonalizedDocumentsRequest:
+      type: object
+      required: [history]
+      properties:
+        history:
+          $ref: './schemas/document.yml#/History'
+        count:
+          $ref: '#/components/schemas/Count'
+        published_after:
+          $ref: './schemas/time.yml#/PublicationDate'
+    StatelessPersonalizedDocumentsResponse:
+      type: object
+      required: [documents]
+      properties:
+        warnings:
+          type: array
+          minItems: 0
+          maxItems: 100
+          items:
+            type: string
+        documents:
+          type: array
+          minItems: 0
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/PersonalizedDocumentData'
```

Comment:
:art: we could define a schema for this array as well as it is used in multiple places.